<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_</class>
 <widget class="QDialog" name="register_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size:12px</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="title">
    <string>用户注册</string>
   </property>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toLoginButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    color: black;
    background-color: transparent;
    box-shadow: none;
    padding: 0;
    border: 0;
	font-size:12px;</string>
    </property>
    <property name="text">
     <string>已有账号？立即登录&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>229</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="acLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>229</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>229</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>229</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:12px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
